transcom/mymove
# INT-B-20465 Customer selecting a counseling office.

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20465)

## Summary
When a customer selects an origin duty location assigned to government counseling, then they will also be asked to select a counseling office within the same GBLOC. This field will be called "Counseling Office".



### How to test as a Customer
1.  Login as a Customer
2. Select a counseling Office while creating a new move
3. Check the Moves data table for corresponding counseling_office_id

### How to test as a Service Counselor
1.  Login as a Service Counselor
2. Verify the counseling queue's counseling Office column matches what was created in previous move


## 508 Screenshots
[508Screenshots-20465.pdf](https://github.com/user-attachments/files/16971315/508Screenshots-20465.pdf)
